packages/datadog_common_test/lib/src/data_helpers.dart

```diff
@@ -21,4 +21,15 @@ extension RandomExtension<T> on List<T> {
   T randomElement() {
     return this[_random.nextInt(length)];
   }
-}
\ No newline at end of file
+}
+
+extension DurationHelpers on Duration {
+  /// The number of whole nanoseconds spanned by this [Duration].
+  ///
+  /// Note, Dart only has precision up to the microsecond level, so the last
+  /// digits of this value will always be zero.
+  /// ```
+  int get inNanoseconds {
+    return inMicroseconds * 1000;
```

Comment:
This one needs to be changed too

Reply:
... No I think this one is right? (1 Microsecond = 1000 Nanosecond)